README.md

```diff
@@ -1,2 +1,80 @@
-# otel
-OpenTelemetry instrumentation library.
+# @fastify/otel
+
+[![CI](https://github.com/fastify/otel/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/fastify/otel/actions/workflows/ci.yml)
+
+<!-- [![NPM version](https://img.shields.io/npm/v/@fastify/otel.svg?style=flat)](https://www.npmjs.com/package/fastify-plugin) -->
+
+[![neostandard javascript style](https://img.shields.io/badge/code_style-neostandard-brightgreen?style=flat)](https://github.com/neostandard/neostandard)
```

Comment:
In my opinion, there's no need to perpetuate these badge things on new repos. For the most part, they were designed to show CI statuses. We use GHA, so such statuses are directly integrated into the repo already (the branch will have a green or red thing with a direct link to the results).

Reply:
SGTM, I mostly took this directly from `fastify-plugin` to keep some sort of standard across the `README`; agree with you, I'll remove them away